packages/evm/package.json

```diff
@@ -55,8 +53,7 @@
         "ts-jest": "^29.0.5",
         "ts-node": "10.9.1",
         "typedoc": "^0.24.1",
-        "typedoc-plugin-markdown": "^3.15.1",
-        "typescript": "5.0.4"
+        "typedoc-plugin-markdown": "^3.15.1"
```

Comment:
are we using this? this was added to get md docs from typedoc annotation.

Reply:
we will be using this IMO

Reply:
create a gh issue as well to track further documentation